modules/private-clusters-setting-dns-private.adoc

```diff
@@ -6,11 +6,17 @@
 [id="private-clusters-setting-dns-private_{context}"]
 = Setting DNS to private
 
-After you deploy a cluster, you can modify its DNS to use only a private zone.
+After you deploy a cluster, you can modify its DNS to use only a private zone, by modifying the `DNS` custom resource (CR).
+Because the Ingress Controller consults the `DNS` CR definition when it creates `Ingress` objects, when you create or modify `Ingress` objects, only private records are created. This secures the cluster by preventing public DNS records from being created.
```

Comment:
OpenShift doesn't create DNS records for ingress objects.  This text was probably supposed to say "`IngressController` objects".
 
```suggestion
Because the Ingress Operator consults the `DNS` CR definition when it creates DNS records for `IngressController` objects, the Operator only creates DNS records for zones that are specified in the `DNS` CR when you create or modify `IngressController` objects.
```